npm/ux/circuit.tsx

```diff
@@ -18,3 +18,31 @@ export function Circuit(props: { circuit: qviz.Circuit }) {
 
   return <div class="qs-circuit" ref={circuitDiv}></div>;
 }
+
+export function CircuitPanel(props: {
+  title: string;
+  subtitle: string;
+  circuit?: qviz.Circuit;
+  errorHtml?: string;
+}) {
+  return (
+    <div>
+      <div>
+        <h1>{props.title}</h1>
+        <h2>{props.subtitle}</h2>
+      </div>
+      {props.circuit ? <Circuit circuit={props.circuit}></Circuit> : null}
+      <div>
+        Tip: you can generate a circuit diagram for any operation that takes
+        qubits or arrays of qubits as input.
+      </div>
+      <div>
+        {props.errorHtml ? (
```

Comment:
Does an error have any styling on it (e.g. a red box, bold text, etc.)? Or is just a regular div.

Reply:
I just added a CSS class so I could tweak the padding. I didn't want to do any styling because it seemed like a big time sink, though I might like to do that in the future. This is how it looks when a circuit fails now. Not pretty but gets the job done?

![image](https://github.com/microsoft/qsharp/assets/16928427/22e32ed5-79a5-4dc0-b32a-a11f781ab6c7)